src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink_de.properties

```
@@ -62,10 +62,19 @@ main.msg.bug=Eine Ausnahme ist in jlink aufgetreten. Melden Sie in der Java-Bugd
 main.extended.help=Liste der verfügbaren Plug-ins:
 
 main.extended.help.footer=Bei Optionen, die eine <Musterliste> erfordern, ist der Wert eine kommagetrennte\nListe von Elementen, die jeweils eines der folgenden Formate verwenden:\n  <Globales Muster>\n  glob:<Globales Muster>\n  regex:<Regex-Muster>\n  @<Dateiname>, wobei der Dateiname der Name einer Datei mit zu verwendenden Mustern ist,\n              ein Muster pro Zeile\n\n
+main.runtime.image.linking.cap.enabled=aktiviert
+main.runtime.image.linking.cap.disabled=deaktiviert
+main.runtime.image.linking.cap.sect.header=Funktionen:
+main.runtime.image.linking.cap.msg=\      Verknüpfung von Laufzeitimage {0}
 
 error.prefix=Fehler:
 warn.prefix=Warnung:
 
+err.runtime.link.not.linkable.runtime=Dieses JDK unterstützt keine Verknüpfung vom aktuellen Laufzeitimage
+err.runtime.link.jdk.jlink.prohibited=Dieses JDK enthält keine als Package integrierten Module und kann nicht verwendet werden, um ein anderes Image mit dem Modul jdk.jlink zu erstellen
+err.runtime.link.packaged.mods=Dieses JDK enthält keine als Package integrierten Module. "--keep-packaged-modules" wird nicht unterstützt
+err.runtime.link.modified.file={0} wurde geändert
+err.runtime.link.patched.module=Datei {0} nicht im Modulimage gefunden. "--patch-module" wird beim Verknüpfen aus dem Laufzeitimage nicht unterstützt
```

This is an old translation. This has changed with https://bugs.openjdk.org/browse/JDK-8343839. My suggested translation would be:

```suggestion
err.runtime.link.patched.module=jlink Assemblierung vom Laufzeitimage wird nicht unterstützt wenn das Laufzeitimage mit "--patch-module" verändert ist.
```

---

Excluding as we discussed on the JBS issue.